config/redirects.yml

```diff
@@ -230,29 +229,28 @@ redirects:
   "/why-teach": "/blog"
   "/become-an-early-years-teacher": "/early-years-teaching-training"
   "/youre-a-kinesthetic-learner": "/"
-  "/get-help-and-support/premier-plus/get-extra-support-with-premier-plus": "/three-things-to-help-you-get-into-teaching"
-  "/premier-plus": "/three-things-to-help-you-get-into-teaching"
+  "/get-help-and-support/premier-plus/get-extra-support-with-premier-plus": "/help-and-support"
+  "/premier-plus": "/help-and-support"
   "/explore-my-options/secondary/teach-chemistry": "/train-to-be-a-teacher"
   "/return-to-teaching": "/returning-to-teaching"
   "/explore-my-options/training/university-led-undergraduate-training": "/train-to-be-a-teacher"
   "/funding-and-salary(*section)": "/funding-and-support"
   "/regional-make-a-difference-in-(*area)": "/"
-  "/how-to-apply(*method)": "/train-to-be-a-teacher/how-to-apply-for-teacher-training"
```

Comment:
the redirects from the old /how-to-apply section on the legacy site seemed to be interfering with the build of the new /how-to-apply section - I've tweaked a lot of the redirects and have also removed this one - just want to check this all looks good to you @plisovin?

Reply:
@EllieNodder `/how-to-apply` gives a 404 now that you have changed the URL of the new section

I think you probably need to add this redirect back in again (pointing to the new section) but it will need tweaking

I suspect that the best you could do it:

"/how-to-apply": "/how-to-apply-for-teacher-training"

because if you add a wildcard, it will stop any other pages in that section being accessible?